examples/verifying_mfd.py

```diff
@@ -1,16 +1,183 @@
+          "(assuming euclidian distance), the maximum distance between "
+          "values is 140 - 135 = 5, which exceeds provided "
+          "parameter of δ = 3. When we tried to verify the same MFD on this "
+          "dataset with δ = 5, this cluster adhered to the MFD as 5 <= δ, "
+          "and the whole MFD held.")
+    print("Romeo and Juliet had the exact same issue- the biggest "
+          "difference between durations of different versions appeared to "
+          "have been 165 - 160 = 5, which held with bigger parameter value.")
+    print("-"*80)
+
+def mfd_strings(table="examples/datasets/addresses_names.csv",
+                bigrams_table="examples/datasets/bigrams.csv"):
+    METRIC = "cosine"
+    METRIC_ALGORITHM = "brute"
+    LHS_INDICES = [1]
+    RHS_INDICES = [2]
+    PARAMETERS = [0.75, 0.5, 0.25, 0.1, 0.01]
+
+    print("Let's showcase this by checking the addresses_names.csv and "
+          "trying to verify the metric functional dependency in it.\n")
+
+    data = pd.read_csv(table, header=[0])
+    print(data)
+
+    for value in PARAMETERS:
+        algo = desbordante.mfd_verification.algorithms.Default()
+        algo.load_data(table=(table, ",", True))
+        algo.execute(lhs_indices=LHS_INDICES, metric=METRIC,
+                                parameter=value, rhs_indices=RHS_INDICES,
+                                metric_algorithm=METRIC_ALGORITHM)
+        print("Checking whether [Movie] -> [Duration] AFD holds with "
+              f"parameter δ = {value}")
+        print_results(algo, data, value)
+
+    print("To get into the intricacies of how the distance has been "
+          "calculated here, we need to first define what is known as an "
+          f"n-gram. {BLUE_CODE}N-grams{DEFAULT_COLOR_CODE} are collections "
+          "of adjacent symbols of fixed "
+          "size, such as 'aab' or 'ba'. Depending on the length, they are "
+          "called bigrams (2), trigrams (3) and so on.\n")
+    print("To turn this definition into a useful tool, let's make a table "
+          "containing a row for every row in addresses_names.csv, and a "
+          "column for every bigram occuring in that table. The number of "
+          "occurences will be written as a value in (row;bigram). "
+          "Our example would contain two rows and 30 columns:")
+
+    bigrams = pd.read_csv(bigrams_table, header=[0])
+    print(bigrams)
+
+    print("If you were to plot two vectors in a 30-dimensional space, and "
+          "then calculate an angle between those two vectors, the cosine of "
+          "that angle would be a cosine distance on bigrams.")
+    print("This algorithm builds on top of that simple principle, but the "
+          "gist of it is that the difference in occurences of bigrams leads "
+          "to those strings being perceived as 'different', or, in metric "
+          "sense, 'distant', providing us with a way of measuring how much "
```

Comment:
```suggestion
          "terms, 'distant', providing us with a way of measuring how much "
```

Reply:
Fixed

Reply:
You can press "Resolve conversation" if there are no additional comments.

Reply:
Some PR guidelines have "reviewer resolves comments" rule in place, so commented the fixes there